Library/ViewModels/SettingsAccountViewModel.swift

```diff
@@ -78,7 +77,7 @@ SettingsAccountViewModelOutputs, SettingsAccountViewModelType {
     self.dismissCurrencyPicker = self.dismissPickerTapProperty.signal
 
     self.transitionToViewController = self.selectedCellTypeProperty.signal.skipNil()
-      .map { SettingsAccountViewModel.viewController(for: $0) }
+      .map(viewControllerFactory)
```

Comment:
We should maybe consider just having the `cellType` returned in the output and keeping `viewController(for cellType:)` in the view controller. The benefit to keeping it in the view model though is that you can test that tapping a particular cell will transition to a VC of a specific type.